src/api/app/models/staging_project.rb

```diff
@@ -0,0 +1,142 @@
+# The staging project is the place where we collect a selection of packages and test, build and review them.
+class StagingProject < Project
```

Comment:
[nitpick] I'm still in favor of introducing a global namespace ``Staging`` and put everything staging workflow related into it:

```
Staging::Project
Staging::Workflow
Staging::RequestsController
Staging::IgnoredRequestsController
etc...
```

Reply:
Hm, not sure if that couldn't collide in some way.

In the end we would have to write something like, unless we rename it:

```ruby
module Staging
  class Project < ::Project
```

Reply:
@bgeuken I don't think this would be a problem?

Reply:
We named it

```ruby
module Staging
  class StagingProject < Project
```
now.